.github/workflows/pre-release.yml

```diff
@@ -0,0 +1,69 @@
+name: Draft release pull request
+
+on:
+    workflow_dispatch:
+        inputs:
+            version:
+                description: "Select version bump"
+                default: "minor"
+                type: choice
+                options:
+                    - major
+                    - minor
+                    - patch
+                    - alpha
+                    - beta
+                    - preview
+                    - dev
+
+jobs:
+    generate-changelog:
+        name: Create a PR to update version and release notes
+        runs-on: sfdc-ubuntu-latest
+        outputs:
+            markdown: ${{ steps.changelog.outputs.markdown }}
+        steps:
+            - uses: actions/checkout@main
+            - name: Set up Python 3.8
+              uses: actions/setup-python@v4
+              with:
+                  python-version: 3.8
+                  cache: pip
+            - name: Install build tool
+              run: python -m pip install hatch
+            - name: Bump version
+              run: hatch version $VERSION
+              env:
+                  VERSION: ${{ inputs.version }}
+            - name: Generate release notes
+              id: changelog
+              env:
+                  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+              run: |
+                  PREVIOUS_VERSION=$(gh release view --json tagName --jq .tagName)
+                  NEXT_VERSION="v$(hatch version)"
+                  echo "## $NEXT_VERSION ($(date -I))" > changelog.md
+                  gh api \
+                    --method POST \
+                    -H "Accept: application/vnd.github.v3+json" \
+                    /repos/jstvz/CumulusCI/releases/generate-notes \
```

Comment:
@jstvz Filesystem reference?

Reply:
Thanks, fixed.

(Left a reference to my testing fork.)